src/graph.rs

```diff
@@ -5117,6 +5281,16 @@ where
   serializer.serialize_u32(source.len() as u32)
 }
 
+fn serialize_source_bytes<S>(
```

Comment:
I don't understand this - is this supposed to serialize the bytes of the source length?

Reply:
It serializes the length instead of the content. It's what the other module types do too so it's patterning off that.

Reply:
That's fine just a confusing name.